301-eventHub-create-authrule-namespace-and-eventHub/README.md

```diff
@@ -0,0 +1,5 @@
+# 301-eventHub-create-authrule-namespace-and-eventHub
```

Comment:
Need a better name rather than just template folder name here. Also is there a reason folder name isn't all lower-case?